src/measure_theory/integral/circle_integral_transform.lean

```diff
@@ -0,0 +1,173 @@
+/-
+Copyright (c) 2022 Chris Birkbeck. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Chris Birkbeck
+-/
+import analysis.complex.cauchy_integral
+import analysis.analytic.basic
+import analysis.calculus.parametric_interval_integral
+import data.complex.basic
+import measure_theory.integral.circle_integral
+/-!
+# Circle integral transform
+
+In this file we define the circle integral transform of a function `f` with complex domain. This is
+defined as $(2πi)^{-1}\frac{f(x)}{x-w}$ where `x` moves along a circle. We then prove some basic
+facts about these functions.
+
+These results are useful for proving that the uniform limit of a sequence of holomorphic functions
+is holomorphic.
+
+-/
+
+open set measure_theory metric filter function
+open_locale interval real
+
+noncomputable theory
+
+variables {E : Type} [normed_group E] [normed_space ℂ E] (R : ℝ) (z w : ℂ)
+
+namespace complex
+
+/-- Given a function `f : ℂ → E`, this gives the function $(2πi)^{-1}\frac{f(x)}{x-w}$ where `x`
+runs over a circle of radius `R` around `z`. If `f` is differentiable and `w` is in the interior of
+the ball, then the integral from `0` to `2 * π` of this gives the value `f(w)`. -/
+def circle_integral_transform (f : ℂ → E) : ℝ → E :=
+  λ θ, (2 * ↑π * I)⁻¹ • deriv (circle_map z R) θ • ((circle_map z R θ) - w)⁻¹ • f (circle_map z R θ)
+
+/-- The derivative of `circle_integral_transform` w.r.t `w` -/
+def circle_integral_transform_deriv (f : ℂ → E) (θ : ℝ) : E :=
+  (2 * ↑π * I)⁻¹ • deriv (circle_map z R) θ • ((circle_map z R θ - w) ^ 2)⁻¹ • f (circle_map z R θ)
+
+lemma circle_integral_transform_deriv_periodic (f : ℂ → E) :
+  periodic (circle_integral_transform_deriv R z w f) (2 * π) :=
+begin
+  have := periodic_circle_map,
+  simp_rw periodic at *,
+  intro x,
+  simp_rw [circle_integral_transform_deriv, this],
+  congr' 2,
+  simp [this],
+end
+
+lemma circle_integral_transform_deriv_eq (f : ℂ → E) :
+  circle_integral_transform_deriv R z w f =
+  (λ θ, (circle_map z R θ - w)⁻¹ • (circle_integral_transform R z w f θ)) :=
+begin
+  ext,
+  simp_rw [circle_integral_transform_deriv, circle_integral_transform, ←mul_smul, ←mul_assoc],
+  ring_nf, simp,
+end
+
+lemma circle_integral_transform_circle_int [complete_space E] (f : ℂ → E) :
+  ∫ (θ : ℝ) in 0..2 * π, circle_integral_transform R z w f θ =
+  (2 * ↑π * I)⁻¹ • ∮ z in C(z, R), (z - w)⁻¹ • f z :=
+begin
+  simp_rw [circle_integral_transform, circle_integral, deriv_circle_map, circle_map],
+  simp,
+end
+
+lemma circle_integral_transform_continuous {R : ℝ} (hR : 0 < R) {f : ℂ → E} {z w : ℂ}
```

Comment:
In general, we try to make sure that the order of symbols in the lemma name matches the order of symbols in the statement, as it makes names easier to guess. Here, `continuous` comes first in the statement, so I'd go for `continuous_circle_transform`. Same thing in a lot of other lemmas.